src/ViewInstance.js

```diff
@@ -406,12 +406,12 @@ class ViewInstance extends React.Component {
     const canMoveItems = stripes.hasPerm('ui-inventory.item.move');
     const canCreateMARCHoldings = stripes.hasPerm('ui-quick-marc.quick-marc-holdings-editor.create');
     const canMoveHoldings = stripes.hasPerm('ui-inventory.holdings.move');
-    const canEditMARCRecord = stripes.hasPerm('records-editor.records.item.put');
-    const canDeriveMARCRecord = stripes.hasPerm('records-editor.records.item.post');
+    const canEditMARCRecord = stripes.hasPerm('ui-quick-marc.quick-marc-editor.all');
+    const canDeriveMARCRecord = stripes.hasPerm('ui-quick-marc.quick-marc-editor.duplicate');
     const hasReorderPermissions = stripes.hasPerm('ui-requests.create') || stripes.hasPerm('ui-requests.edit') || stripes.hasPerm('ui-requests.all');
+    const canViewMARCSource = stripes.hasPerm('ui-quick-marc.quick-marc-editor.view');
 
-    const canCreateMARCHoldingsForInstanceWithSourceMARC = isSourceMARC && canCreateMARCHoldings;
-    const canEditDeriveMARCRecord = isSourceMARC && (canEditMARCRecord || canDeriveMARCRecord);
+    const showQuickMarcMenuSection = (isSourceMARC && canCreateMARCHoldings) || (isSourceMARC && (canEditMARCRecord || canDeriveMARCRecord));
```

Comment:
This can be simplified a bit: `isSourceMARC && (canCreateMARCHoldings || canEditMARCRecord || canDeriveMARCRecord)`